configure.ac

```diff
@@ -626,9 +626,6 @@ case $host in
      ARCH="aarch64"
      use_arch="aarch64"
      use_neon=yes
-     use_gles=yes
```

Comment:
can you please add a test that keeps this as default, if the user didn't specify --enable-gles or --enable-gl?